sqls/release-analysis/sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+    repo_id,
+    COUNT(*) AS total_releases_count,
+    MAX(release_published_at) AS last_release_time,
+    365 / COUNT(*) AS average_interval,
+    AVG(LENGTH(release_body)) AS average_body_length
+FROM
+    {{table}}
```

Comment:
The release analysis should be a case study SQL component which should be in case study folder and the result should be for every single repo but not in a total aggregation.

Reply:
My bad, actually you `GROUP BY repo_id` will split into different repos, but without `WHERE` clause you can not get correct result of release status. `COUNT(*)` will count all log counts of a repo in a year.

Reply:
I got what you mean, I think I still have no enough insight on the database structure. So I directly mistake all records for release item. Should I close this PR or keep on fix this? since this feature are included on another component.

Reply:
Can I close this PR and add the function myself?

Reply:
Yes and sorry for all the inconvenience and delay I have caused.

Reply:
No problem.